Riot/Modules/Room/Views/BubbleCells/RoomTypingBubbleCell.swift

```diff
@@ -38,9 +42,7 @@ class RoomTypingBubbleCell: UITableViewCell {
     override func awakeFromNib() {
         super.awakeFromNib()
         
-        additionalUsersLabel?.textColor = ThemeService.shared().theme.textSecondaryColor
-        dotsView?.highlightedDotColor = ThemeService.shared().theme.textTertiaryColor
-        dotsView?.dotColor = ThemeService.shared().theme.textSecondaryColor
+        update(theme: ThemeService.shared().theme)
```

Comment:
It's probably better to call `update(theme:)` from `tableView:cellForRowAtIndexPath:` method as we do across the app. And we can drop this and the one in `prepareForReuse`.